bidwire/scrapers/boston_notice_scraper.py

```diff
@@ -0,0 +1,68 @@
+import re
+import logging
+import scrapelib
+from document import Document
+from datetime import datetime
+from datetime import date as dtdate
+from .base_scraper import BaseScraper
+from lxml import html
+from concurrent.futures import ThreadPoolExecutor
+
+
+# Logger object for this module
+log = logging.getLogger(__name__)
+
+
+class NoticesScraper(BaseScraper):
```

Comment:
Please make the naming of file, class and enum as aligned as possible. If the file is `boston_notice_scraper.py`, let's make the class `BostonNoticeScraper`, and the enum `Document.Site.BOSTON_NOTICES`.